src/observable/FromEventObservable.ts

```diff
@@ -133,6 +133,8 @@ export class FromEventObservable<T, R> extends Observable<T> {
       const source = sourceObj;
       sourceObj.addListener(eventName, handler);
       unsubscribe = () => source.removeListener(eventName, handler);
+    } else {
+      throw new TypeError('Invalid event target');
```

Comment:
While there are some of mixtures around, in majority error messages are not capitalized. May need to align existing codebase as well.